CHANGELOG.md

```diff
@@ -38,6 +40,33 @@ If we missed any change or there's something you'd like to discuss about this ve
     + `Spotify::prompt_for_user_token[_without_cache]`
     + The `ClientError::CLI` variant, for whenever user interaction goes wrong
 - Fix typo in `user_playlist_remove_specific_occurrenes_of_tracks`, now it's `user_playlist_remove_specific_occurrences_of_tracks`.
+- All fallible calls in the client return a `ClientError` rather than using `failure`.
+- Endpoints take `Vec<String>/&[String]` as parameter have changed to `impl IntoIterator<Item = &str>`, which is backward compatibility.
+  + The endpoints which changes parameter from `Vec<String>` to `impl IntoIterator<Item = &str>`:
+	- `artists`
+	- `albums`
+	- `save_shows`
+	- `get_several_episodes`
+	- `check_users_saved_shows`
+	- `remove_users_saved_shows`
+  + The endpoints which changes parameter from `&[String]` to `impl IntoIterator<Item = &str>`:
+	- `user_playlist_add_tracks`
+	- `user_playlist_replace_tracks`
+	- `user_playlist_remove_all_occurrences_of_tracks`
+	- `current_user_saved_tracks_delete`
+	- `current_user_saved_tracks_contains`
+	- `current_user_saved_tracks_add`
+	- `current_user_saved_albums_add`
+	- `current_user_saved_albums_delete`
+	- `current_user_saved_albums_contains`
+	- `user_follow_artists`
+	- `user_unfollow_artists`
+	- `user_artist_check_follow`
+	- `user_follow_users`
+	- `user_unfollow_users`
+	- `audios_features`
```

Comment:
Now that I realize, `audios_features` should be renamed to `audio_features`. Should we do that in this PR or leave it for later?

Reply:
I could fix it in this PR.

Reply:
sure, that's probably easier

Reply:
Well, it turns out that it's not a problem, we have both `audio_features` and `audios_features` endpoint:

+ audio_features: Get audio features for a track
+ audios_features: Get Audio Features for Several Tracks

Reply:
Ah ok. How about `track` instead of `audio`? "audios" seems off to me, perhaps `track_features` and `tracks_features` would be more fitting

Reply:
Yep, it's a more suitable name, I will rename `audio_features`, `audios_features` and `audio_analysis` to `track_features`, `tracks_features` and `track_analysis`.